index.js

```diff
@@ -23,21 +26,63 @@ Object.keys(ansiStyles).forEach(function (key) {
 
 	styles[key] = {
 		get: function () {
-			return build.call(this, this._styles ? this._styles.concat(key) : [key]);
+			var codes = ansiStyles[key];
+			return build.call(this, this._styles ? this._styles.concat(codes) : [codes], key);
+		}
+	};
+});
+
+ansiStyles.color.closeRe = new RegExp(escapeStringRegexp(ansiStyles.color.close), 'g');
```

Comment:
I mean, it's a little bit dangerous to assign to `ansiStyles` as it's a single-ton. So if someone uses `ansi-styles` another place, their instance will also be monkey-patched. Might be a non-issue in practise, but wanted to bring it up.